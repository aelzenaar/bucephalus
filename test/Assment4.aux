\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\providecommand \oddpage@label [2]{}
\babel@aux{english}{}
\@writefile{toc}{\contentsline {section}{\tocsection {}{}{Question One}}{1}{section*.1}}
\@writefile{toc}{\contentsline {paragraph}{\tocparagraph {}{}{(a)}}{1}{section*.2}}
\@writefile{toc}{\contentsline {paragraph}{\tocparagraph {}{}{(b)}}{1}{section*.3}}
\@writefile{toc}{\contentsline {section}{\tocsection {}{}{Question Two}}{1}{section*.4}}
\@writefile{toc}{\contentsline {section}{\tocsection {}{}{Question Thre}}{2}{section*.5}}
\@writefile{toc}{\contentsline {paragraph}{\tocparagraph {}{}{(a)}}{2}{section*.6}}
\@writefile{toc}{\contentsline {paragraph}{\tocparagraph {}{}{(b)}}{2}{section*.7}}
\@writefile{toc}{\contentsline {paragraph}{\tocparagraph {}{}{(c)}}{2}{section*.8}}
\@writefile{toc}{\contentsline {section}{\tocsection {}{}{Question Four}}{2}{section*.9}}
\newlabel{tocindent-1}{0pt}
\newlabel{tocindent0}{12.78pt}
\newlabel{tocindent1}{0pt}
\newlabel{tocindent2}{0pt}
\newlabel{tocindent3}{0pt}
\@writefile{toc}{\contentsline {section}{\tocsection {}{}{Question Five}}{3}{section*.10}}
