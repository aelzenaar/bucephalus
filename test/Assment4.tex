{"Buc_author":"Alexander Elzenaar", "Buc_hp":"asst",
 "Buc_title":"MATHS 326 assignment 4",
 "duedate": "Thursday 24 May 2018", "studentid":"200878696",
 "semester": "S1 2018", "course":"MATHS 326", "upi": "\\MakeLowercase{aelz176}",
 "asstno":"4", "Buc_tags": ["homework", "auckland", "mathematics", "combinatorics", "326"]}
===
\section*{Question One}
\paragraph{(a)}
The symmetry group of the bracelet is $ C_6 $.
\begin{center}\begin{tabular}{|c|c|c|}\hline
  \textbf{Cycle type} & \textbf{Count} & \textbf{Fixed colourings}\\\hline
  $[1^6]$ & 1 & $ 2^6 $\\
  $[2^3]$ & 1 & $ 2^3 $\\
  $[3^2]$ & 2 & $ 2^2 $\\
  $[6]$ & 2 & $ 2^1 $\\\hline
   & 6 &\\\hline
\end{tabular}\end{center}

So the number of orbits of $ C_6 $ on the colourings is
\begin{displaymath}
  \frac{1 \cdot 2^6 + 1 \cdot 2^3 + 2 \cdot 2^2 + 2 \cdot 2^1}{6} = 14.
\end{displaymath}

There are 14 distinct colourings up to rotation.

\paragraph{(b)}
If we also include reflections, then the symmetry group of the bracelet is $ D_6 $ (of order 12).
\begin{center}\begin{tabular}{|c|c|c|}\hline
  \textbf{Cycle type} & \textbf{Count} & \textbf{Fixed colourings}\\\hline
  $[1^6]$ & 1 & $ 2^6 $\\
  $[2^3]$ & 1 & $ 2^3 $\\
  $[3^2]$ & 2 & $ 2^2 $\\
  $[6]$ & 2 & $ 2^1 $\\\hline
  $[1^2,2^2] $ & 3 & $ 2^4 $\\
  $[2^3] $ & 3 & $ 2^3 $\\\hline
   & 12 &\\\hline
\end{tabular}\end{center}

So the number of orbits of $ D_6 $ on the colourings is
\begin{displaymath}
  \frac{1 \cdot 2^6 + 1 \cdot 2^3 + 2 \cdot 2^2 + 2 \cdot 2^1 + 3 \cdot 2^4 + 3 \cdot 2^3}{12} = 13.
\end{displaymath}

There are 13 distinct colourings up to rotation and reflection.

\section*{Question Two}
The cube's symmetry group has order 24.
\begin{center}\begin{tabular}{|c|c|c|}\hline
  \textbf{Cycle type on edges} & \textbf{Count} & \textbf{Fixed colourings}\\\hline
  $[1^{12}]$ & 1 & $ 2^{12} $\\
  $[4^3]$ & 6 & $ 2^3 $\\
  $[2^6]$ & 3 & $ 2^6 $\\
  $[3^4]$ & 8 & $ 2^4 $\\
  $[1^2,2^5]$ & 6 & $ 2^7 $\\\hline
   & 24 &\\\hline
\end{tabular}\end{center}

So the number of orbits of the group on the edge colourings is
\begin{displaymath}
  \frac{1 \cdot 2^{12} + 6 \cdot 2^3 + 3 \cdot 2^6 + 8 \cdot 2^4 + 6 \cdot 2^7}{24} = 218.
\end{displaymath}

There are 218 distinct colourings up to rotation.

\section*{Question Thre}
\paragraph{(a)}
Labelling the vertices with the numbers 1 to 5, and writing the edge between vertex 1 and vertex 2 by 12, we have the following:
\begin{gather*}
  12 \mapsto 12\\
  13 \mapsto 24 \mapsto 15 \mapsto 23 \mapsto 14 \mapsto 25 \mapsto 13\\
  34 \mapsto 45 \mapsto 53.
\end{gather*}

\paragraph{(b)}
The symmetry group of $ K_5 $ is $ S_5 $. We have the following:
\begin{center}\begin{tabular}{|c|c|c|c|}\hline
  \textbf{Cycle type on vertices} & \textbf{Count} & \textbf{Cycle type on edges} &\textbf{Fixed colourings}\\\hline
  $ [1^5] $ & 1 & $ [1^{10}] $ & $ k^{10} $\\
  $ [1^3,2] $ & 10 & $ [1^4,2^3] $ & $ k^7 $\\
  $ [1^2,3] $ & 20 & $ [1,3^3] $ & $ k^4 $\\
  $ [1, 4] $ & 30 & $ [2,4^2] $ & $ k^3 $\\
  $ [1,2,2] $ & 15 & $ [1^2,4^2] $ & $ k^4 $\\
  $ [2,3] $ & 20 & $ [1,3,6] $ & $ k^3 $\\
  $ [5] $ & 24 & $ [5^2] $ & $ k^2 $\\\hline
  & 120 &&\\\hline
\end{tabular}\end{center}

Hence the cycle index of the symmetry group is
\begin{displaymath}
  \frac{1}{120}(a_1^{10} + 10a_1^2 a_2^3 + 20a_1 a_3^3 + 30a_2 a_4^2 + 15a_1^2 a_4^2 + 20 a_1 a_3 a_6 + 24a_5^2).
\end{displaymath}

\paragraph{(c)}
Counting the number of non-isomorphic graphs on five vertices is equivalent to counting the number of 2-colourings of $ K^5 $.
Hence we need only calculate
\begin{displaymath}
  \frac{1}{120}(2^{10} + 10(2^5) + 20(2^4) + 30(2^3) + 15(2^4) + 20(2^3) + 24(2^2)) = 20.
\end{displaymath}

There are 20 distinct graphs on five vertices up to isomorphism.

\section*{Question Four}
The symmetry group of the pentagonal prism is $ C_2 \times C_5 $.
\begin{center}\begin{tabular}{|c|c|c|}\hline
  \textbf{Cycle type} & \textbf{Count} & \textbf{Cycle monomial}\\\hline
  $ [1^7] $ & 1 & $ a_1^7 $\\
  $ [1^2,5] $ & 4 & $ a_1^2 a_5 $\\
  $ [1^5,2] $ & 1 & $ a_1^5 a_2 $\\
  $ [2,5] $ & 4 & $ a_2 a_5 $\\\hline
   & 10 &\\\hline
\end{tabular}\end{center}

So the cycle index on the face colourings is
\begin{displaymath}
  \frac{1}{10}(a_1^7 + 4a_1^2 a_5 + a_1^5 a_2 + 4a_2 a_5).
\end{displaymath}
Let $ x $, $ y $, and $ z $ be the number of green, blue, and red edges respectively; so the generating function for
the colorings of a particular edge is $ f(x,y,z) = x + y + z $. Hence, we want the coefficient of $ x^2 y^2 z^2 $ in
\begin{displaymath}
  \begin{split}
    \frac{1}{10}((x + y + z)^7 + 4(x + y + z)^2 (x^5 + y^5 + z^5) &+ (x + y + z)^5 (x^2 + y^2 + z^2)\\
    &{}+ 4(x^2 + y^2 + z^2)(x^5 + y^5 + z^5)).
  \end{split}
\end{displaymath}
This coefficient is
\begin{displaymath}
  \frac{1}{10}\left(\frac{7!}{2!2!2!} + 0 + 3 \cdot \frac{5!}{2!2!0!} + 0\right) = 72.
\end{displaymath}

There are 72 distinct colourings such that each colour appears precisely twice, up to rotation.

\section*{Question Five}
The symmetry group of the dodecahedron has order sixty.
\begin{center}\begin{tabular}{|c|c|c|}\hline
  \textbf{Cycle type} & \textbf{Count} & \textbf{Cycle monomial}\\\hline
  $ [1^{12}] $ & 1 & $ a_1^{12} $\\
  $ [1^2,5^2] $ & 24 & $ a_1^2a_5^2 $\\
  $ [2^6] $ & 15 & $ a_2^6 $\\
  $ [3^4] $ & 20 & $ a_3^4 $\\\hline
   & 60 &\\\hline
\end{tabular}\end{center}

So the cycle index on the face colourings is
\begin{displaymath}
  \frac{1}{60}(a_1^{12} + 24a_1^2a_5^2 + 15a_2^6 + 20a_3^4).
\end{displaymath}
Let $ x $, $ y $, and $ z $ be the number of red, green, and white faces respectively; so the generating function for
the colorings of a particular face is $ f(x,y,z) = x + y + z $. Hence, we want the coefficient of $ x^3y^3z^6 $ in
\begin{displaymath}
  \frac{1}{60}\left((x+y+z)^{12} + 24(x+y+z)^2(x^5+y^5+z^5)^2 + 15(x^2+y^2+z^2)^6 + 20(x^3+y^3+z^3)^4\right).
\end{displaymath}
This coefficient is
\begin{displaymath}
  \frac{1}{60}\left(\frac{12!}{3!3!6!} + 0 + 0 + 20\cdot\binom{4}{1}\cdot\binom{3}{1}\cdot\binom{2}{2}\right) = 312.
\end{displaymath}
