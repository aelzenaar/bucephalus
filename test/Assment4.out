\BOOKMARK [1][-]{section*.1}{Question\040One}{}% 1
\BOOKMARK [1][-]{section*.4}{Question\040Two}{}% 2
\BOOKMARK [1][-]{section*.5}{Question\040Thre}{}% 3
\BOOKMARK [1][-]{section*.9}{Question\040Four}{}% 4
\BOOKMARK [1][-]{section*.10}{Question\040Five}{}% 5
